services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkMaintenanceSpec.groovy

```diff
@@ -11,19 +12,9 @@ import org.openkilda.functionaltests.helpers.Wrappers
 import org.openkilda.messaging.info.event.IslChangeType
 import org.openkilda.messaging.info.event.PathNode
 import org.openkilda.messaging.payload.flow.FlowState
-import org.openkilda.testing.Constants
-
-import org.springframework.beans.factory.annotation.Value
 
 class LinkMaintenanceSpec extends HealthCheckSpecification {
 
-    @Value('${isl.cost.when.under.maintenance}')
-    int islCostWhenUnderMaintenance
```

Comment:
If this and `isl.cost.when.port.down` no longer exist, then it should be removed from all the templates and example properties

Reply:
in fact we still use this value during path computation.
we add this value to cost in real time and don't update cost for ISL in db